app/kuma-dp/pkg/dataplane/meshmetrics/component.go

```diff
@@ -68,6 +68,8 @@ func (cf *ConfigFetcher) Start(stop <-chan struct{}) error {
 		"socketPath", fmt.Sprintf("unix://%s", cf.socketPath),
 	)
 
+	ctx := context.Background()
```

Comment:
Should this be canceled when `stop` is closed?

Reply:
you are right, should be good now

Reply:
@Automaat I think it needs to be closed in a separate go routine, if we're executing `cf.configureOpenTelemetryExporter`, we can't react to the channel closing in `case <-stop:` I don't think